README.md

```diff
@@ -9,5 +12,53 @@ to sinks such as Kudu, HBase, and Database.
 * [Oracle](docs/oracle/Oracle.md)
 * [Microsoft SQL Server](docs/CTSQLServer.md)
 
+# Overview
+
+Following plugins are available in this repository. 
+
+  * [Google Cloud BigTable Sink](docs/CDCBigTable-sparksink.md)
+  * [HBase Sink](docs/CDCHBase-sparksink.md)
+  * Kudu Sink 
+  * [Golden Gate Kafka Source](docs/oracle/Oracle.md)
+  * [SQL Server Change Tracking Streaming Source](docs/CTSQLServer.md)
+
+# Contact
+
+## Mailing Lists
+
+CDAP User Group and Development Discussions:
+
+* [cdap-user@googlegroups.com](https://groups.google.com/d/forum/cdap-user)
+
+The *cdap-user* mailing list is primarily for users using the product to develop
+applications or building plugins for appplications. You can expect questions from
+users, release announcements, and any other discussions that we think will be helpful
+to the users.
+
+## Slack Team
```

Comment:
I don't think many people are monitoring the slack channel anymore, let's remove this section.

Reply:
fixed